lib/swarm/tracker/tracker.ex

```diff
@@ -120,9 +120,13 @@ defmodule Swarm.Tracker do
     # wait for node list to populate
     nodelist = Enum.reject(Node.list(:connected), &ignore_node?/1)
 
-    strategy = Node.self
-           |> Strategy.create()
-           |> Strategy.add_nodes(nodelist)
+    strategy =
+      case ignore_node?(Node.self) do
```

Comment:
Can you explain this part of the change in more detail? I don't think it does what you expect it to.